ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Block.hs

```diff
@@ -545,7 +545,7 @@ pattern QueryAnytimeAllegra q = QueryAnytime q (EraIndex (S (S (Z (K ())))))
 pattern QueryAnytimeMary
   :: QueryAnytime result
   -> CardanoQuery c result
-pattern QueryAnytimeMary q = QueryAnytime q (EraIndex (S (S (Z (K ())))))
+pattern QueryAnytimeMary q = QueryAnytime q (EraIndex (S (S (S (Z (K ()))))))
```

Comment:
Yep! Before your diff, this module contains these definitions:

```
pattern BlockByron b          = HardForkBlock (OneEraBlock (Z (I b)))
pattern QueryAnytimeByron q   = QueryAnytime q (EraIndex   (Z (K ())))

pattern BlockShelley b        = HardForkBlock (OneEraBlock (S (Z (I b))))
pattern QueryAnytimeShelley q = QueryAnytime q (EraIndex   (S (Z (K ()))))

pattern BlockAllegra b        = HardForkBlock (OneEraBlock (S (S (Z (I b)))))
pattern QueryAnytimeAllegra q = QueryAnytime q (EraIndex   (S (S (Z (K ())))))

pattern BlockMary b           = HardForkBlock (OneEraBlock (S (S (S (Z (I b))))))
pattern QueryAnytimeMary q    = QueryAnytime q (EraIndex   (S (S (Z (K ())))))    -- <-- breaks the pattern!
```

Thanks for catching this typo!